### PR TITLE
Add variable to conditionally install dependencies

### PR DESCRIPTION
**Pull request by jyaworski**

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -3,6 +3,6 @@ lsyncd_source: /tmp/lsyncd-{{ lsyncd_release }}
 lsyncd_bin: /usr/local/bin/lsyncd
 lsyncd_pid: /var/run/lsync
 lsyncd_service: /etc/systemd/system/lsync.service
-
+lsyncd_install_dependencies: true
```
Comment (ccremer) [inline vars/main.yml]: Remove this line please. This variable is already defined in the defaults\main.yml

File: README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ lsyncd_conf: /etc/lsyncd.conf.lua
 lsyncd_targets: []
 lsyncd_extra_settings: []
 lsyncd_retry: true # the 'insist' option
+lsyncd_install_dependencies: false # install dependencies outside of this role
```
Comment (ccremer) [inline README.md]: It should be set to true, as it's defined in defaults\main.yml

File: tasks/install-debian.yml [hunk 3]
```diff
@@ -10,3 +10,4 @@
     - cmake
     - g++
   become: true
+  when: "{{ lsyncd_install_dependencies }}"
```
Comment (ccremer) [inline tasks/install-debian.yml]: I think boolean variables don't need " and {{ in the when clause, just declare `when: lsyncd_install_dependencies`

File: tasks/install-centos.yml [hunk 4]
```diff
@@ -10,3 +10,4 @@
     - lua-devel
     - g++
   become: true
+  when: "{{ lsyncd_install_dependencies }}"
```
Comment (ccremer) [inline tasks/install-centos.yml]: I think boolean variables don't need " and {{ in the when clause, just declare `when: lsyncd_install_dependencies`

**Review by ccremer — CHANGES_REQUESTED** [review]
Thank you for your contribution! I think it could be improved a bit though

**Comment by jyaworski** [comment]
@ccremer I've addressed your comments and rebased.

**Review by ccremer — APPROVED** [review]

**Pull request merged**